scipy/spatial/tests/test_spherical_voronoi.py

```diff
@@ -170,3 +172,33 @@ def test_single_hemisphere_handling(self):
             dots = np.einsum('ij,ij->i', sv.vertices, triangles[:, 0])
             circumradii = np.arccos(np.clip(dots, -1, 1))
             assert np.max(circumradii) > np.pi / 2
+
+    def test_minimum_dimensionality(self):
+        with assert_raises(ValueError):
+            sv = spherical_voronoi.SphericalVoronoi(np.linspace(0, 1, 10))
+
+    @pytest.mark.parametrize("dim", range(2, 7))
+    def test_higher_dimensions(self, dim):
+        n = 100
+        points = np.random.randn(n, dim)
+        points /= np.linalg.norm(points, axis=1)[:, np.newaxis]
+        sv = SphericalVoronoi(points)
+        assert sv.vertices.shape[1] == dim
+
+    @pytest.mark.parametrize("dim", range(2, 7))
+    def test_cross_polytope_regions(self, dim):
```

Comment:
some brief comments to explain what these cross polytope tests are doing might be helpful--I'm still trying to wrap my head around these--I'll have to check the page you linked more closely

Reply:
I have fixed one of the tests and added some comments.  In brief, these are polytopes whose Voronoi vertex coordinates are known, and which we can test in all dimensions.